src/Engine/Player/Core/Player.cs

```diff
@@ -72,7 +70,7 @@ public async void LoadAssets()
                     textToDisplay += " by " + _scene.Header.CreatedBy;
 
                 if (_scene.Header.CreationDate != null)
-                    textToDisplay += " on " + _scene.Header.CreationDate;
+                    _ = " on " + _scene.Header.CreationDate;
```

Comment:
why `textToDisplay +=` to `_ =`?

Reply:
This is basically an error in code, since `textToDisplay` is never used.